### PR TITLE
All L, U & N reqs done

### PR DESCRIPTION
**Pull request by donaldsawyer**

### CODE REVIEW

File: twtr/grails-app/assets/javascripts/app/controllers/userDetail-controller.js [hunk 1]
```diff
@@ -68,4 +115,7 @@ angular.module('app').controller('userDetailController', function($scope, $locat
                 $scope.getAccount();
             })
     };
+
+    if($scope.auth.account == null)
```
Comment (wang2208) [inline twtr/grails-app/assets/javascripts/app/controllers/userDetail-controller.js:119]: why is this needed since the auth.account was already checked and used previously

Comment (donaldsawyer) [inline twtr/grails-app/assets/javascripts/app/controllers/userDetail-controller.js:119]: On the first time you log in, you have to initialize the authorized user's account in the controller and you have to do it at the end after all the account info had been grabbed.  When the controller is initialized the first time, all that has happened is a login.
